### ISSUE TITLE
Unable to view & enable notifications for private repos

### ISSUE THREAD

**Issue by sst2010** [Daskiworks/ghwatch#127]
- Just logged in to the app on MI Note 5 Pro.
- Granted access to organisation during auth.
- Update org. settings to remove all restrictions.
- Still can't see the org. or non-org. private repo. in Watched Repo. section.
  - Only 2 repos are appearing in this section. They are 2 public repos I already chose to watch from the browser.
  
Any tips to resolve this ??

**Comment by velias** [comment]
https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/about-oauth-app-access-restrictions

**Comment by velias** [comment]
Some users hit this restriction, so I documented it at https://github.com/Daskiworks/ghwatch#notifications-from-github-private-repositories

**Comment by sst2010** [comment]
@velias 

I already checked out the said page & enabled all access for the app from my account & for the organization. Still facing the issue. I'll share a few pics in a bit to show the settings for better clarity.

**Comment by sst2010** [comment]
@velias 

See the below images to confirm the needed account settings.

1.) 
![Full repo. & org. access to the app](https://user-images.githubusercontent.com/25411283/103434501-29e1d180-4c28-11eb-8e63-0fde57ec1e56.png)
2.) 
![No restriction policy in the org.](https://user-images.githubusercontent.com/25411283/103434505-3c5c0b00-4c28-11eb-8ed3-f81f9d18d309.png)

If you need anything else to assess the situation, let me know. Thanks for your time !!

**Comment by velias** [comment]
Hmm, everything looks good to me, I compared it with my profile. But I'm not big expert in these Github settings as I do not manage any private repo and do not use them too much at all :-( 
I read again your initial info, and I'm not sure if I understood it correctly. "Watched Repositories" view shows only repos you started to watch from the browser, this is how it works (you can't add new watched repo through this app). You mentioned there that you started to watch two public repos. Do you watch this private repo also you are asking about (I expect yes, but only to be sure)?

**Comment by sst2010** [comment]
Before I started using the `ghwatch` app, I had previously watched 2 public repos & 2 private repos from the GitHub web app. But in the app, only the public repos appear in the `Watched Repositories` list. I hope that clears your queries.

**Comment by velias** [comment]
Thanks for the clarification. I must say I do not see anything wrong in settings you showed in screenshots.
In your description, it looks like step "Update org. settings to remove all restrictions." is after you authorized the app. Github docs says that apps automatically get access after you remove these restrictions, but maybe you should try to remove access for GH::watch and try to login/authorize it again now when the restrictions are removed from the organization/repo.

**Comment by sst2010** [comment]
@velias 
I tried that already when the issue happened. Although, I recently created some new private repos & began receiving notifications for them. Also, after this happened, I was able to receive notifications about the private repos created before as well.

I can't figure out why it started working all of a sudden days after I changed the settings & reconfigured the app.

Anyways, I think the issue can be closed now unless you wish to investigate this further.
